### PR TITLE
forticlient: use offline installer & rewrite uninstall

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/forticlient.rb [hunk 1]
```diff
@@ -1,22 +1,68 @@
 cask "forticlient" do
-  version "6.4"
-  sha256 "a673cc372f4caf90476e6cda4e51c8450ac58e09a8c5218bba537e86792e5d23"
+  version "6.4.2.1305"
+  sha256 "660901a65c72f06c1c4b93a01cd46e562a684c59e83c0cdfa75b254c506939cc"
 
-  url "https://filestore.fortinet.com/forticlient/downloads/FortiClientVPNOnlineInstaller_#{version}.dmg",
-      verified: "filestore.fortinet.com/forticlient/"
+  url "https://d3gpjj9d20n0p3.cloudfront.net/forticlient/downloads/FortiClient_#{version}_macosx.dmg",
```
Comment (vitorgalvao) [inline Casks/forticlient.rb:5]: How do we get this?

Comment (i0ntempest) [reply Casks/forticlient.rb:5]: Right here on the official website
https://www.forticlient.com/downloads

**Review by vitorgalvao — APPROVED** [review]

**Comment by mxalbert1996** [comment]
This PR completed changed the software this cask installs. Formerly it installs "FortiClient VPN" which is free and now it installs "FortiClient" which is unfree. These are similar but different softwares. The difference can be found here: https://www.forticlient.com/downloads.

**Comment by i0ntempest** [comment]
> This PR completely changed the software this cask installs. Formerly it installs "FortiClient VPN" which is free and now it installs "FortiClient" which is unfree. These are similar but different softwares. The difference can be found here: https://www.forticlient.com/downloads.

My bad for not checking thoroughly before opening the PR, but the cask token here is a bit misleading. Imo FortiClient VPN should use another token. I’ll check if an offline installer exists for that one and make a new cask.

**Pull request merged**